src/net/tnemc/item/SerialItem.java

```diff
@@ -83,7 +83,10 @@ public SerialItem(ItemStack item, Integer slot) {
       display = stack.getItemMeta().getDisplayName();
       lore = stack.getItemMeta().getLore();
 
-      if(stack.getItemMeta().hasCustomModelData()) {
+      // Quick 1.13 patch
+      boolean customModelCompat = !Bukkit.getVersion().contains("1.13");
```

Comment:
Instead of creating a separate variable just put this in the if statement.

Reply:
Fixed